WORKSPACE

```diff
@@ -25,7 +25,7 @@ load("@bazel_skylib//lib:versions.bzl", "versions")
 
 # Keep this version in sync with:
 #  * The BAZEL environment variable defined in .github/workflows/ci.yml, which is used for CI and nightly builds.
-versions.check(minimum_bazel_version = "4.2.2")
+versions.check(minimum_bazel_version = "4.2.2", maximum_bazel_version="5.4.0")
```

Comment:
Let's add a TODO with a tracking GH issue for Bazel 6+ support?

Reply:
Done